relayer/app/config.go

```diff
@@ -16,19 +17,23 @@ import (
 )
 
 type Config struct {
-	RPCEndpoints   xchain.RPCEndpoints
-	PrivateKey     string
-	HaloURL        string
-	Network        netconf.ID
-	MonitoringAddr string
+	RPCEndpoints    xchain.RPCEndpoints
+	PrivateKey      string
+	HaloURL         string
+	Network         netconf.ID
+	MonitoringAddr  string
+	DBDir           string
+	ConfirmInterval time.Duration
```

Comment:
Config is only for actual flags. This can be hardcoded in `NewCursors` or its caller.